docs/ui_configuration.md

```diff
@@ -0,0 +1,88 @@
+# Configuring the UI
+
+The admin has the ability to configure most parts of the UI to some extent.
+
+The Ui configuration section has this structure by default:
+
+```
+ui:
```

Comment:
This does not follow the structure we agreed on in https://github.com/opendatahub-io/jupyterhub-singleuser-profiles/issues/90#issuecomment-782035512